core/audits/dobetterweb/dom-size.js

```diff
@@ -118,6 +121,23 @@ class DOMSize extends Audit {
           value: stats.width.max,
         },
       },
+      {
+        statistic: str_(UIStrings.statisticDOMTotalFrames),
+        value: {
+          type: 'numeric',
+          granularity: 1,
+          value: stats.totalFrames,
+        },
+      },
+      {
+        node: {type: 'url', value: stats.framesDepth.url},
```

Comment:
anyone have a good way to grab the node id of the iframe element a frame is associated with? should we just guess and do a query selector for the same `src` url?

Reply:
I looked around in the CDP docks with no luck. In the meantime I think it's fine to just use the `url`. Maybe rename this key `node` -> `frame` or something as well.

Reply:
You're suggesting introducing another column to the table?

Reply:
Ope didn't realize why this was named `node` in the first place. Probably fine the way it is then.